source/beamline/AMEnumeratedControl.h

```diff
@@ -11,6 +11,9 @@ class AMEnumeratedControl : public AMPseudoMotorControl
     Q_OBJECT
 
 public:
+	/// Enumeration of some default option indices.
+	enum DefaultOptions { Unknown = -1, Invalid = -2 };
```

Comment:
Maybe they are used elsewhere, but I didn't see any use for these enums.  Are they used anywhere?